doc/api/crypto.md

```diff
@@ -766,20 +777,18 @@ is returned.
 added: v0.11.14
 -->
 - `encoding` {string}
-
-Returns the EC Diffie-Hellman private key in the specified `encoding`,
-which can be `'latin1'`, `'hex'`, or `'base64'`. If `encoding` is provided
-a string is returned; otherwise a [`Buffer`][] is returned.
+- Returns: {Buffer | string} The EC Diffie-Hellman private key in the specified
+  `encoding`, which can be `'latin1'`, `'hex'`, or `'base64'`. If `encoding`
+  is provided a string is returned; otherwise a [`Buffer`][] is returned.
 
 ### ecdh.getPublicKey([encoding][, format])
 <!-- YAML
 added: v0.11.14
 -->
 - `encoding` {string}
 - `format` {string} **Default:** `uncompressed`
-
-Returns the EC Diffie-Hellman public key in the specified `encoding` and
-`format`.
+- Returns: {Buffer | string} the EC Diffie-Hellman public key in the specified
```

Comment:
Uppercase for consistency? `{Buffer | string} The`

Reply:
Done in the fixup commit.